python/pyspark/ml/feature.py

```diff
@@ -2612,6 +2612,8 @@ class StopWordsRemover(JavaTransformer, HasInputCol, HasOutputCol, JavaMLReadabl
 
     .. note:: null values from input array are preserved unless adding null to stopWords explicitly.
 
+    >>> locale = spark._jvm.java.util.Locale
+    >>> locale.setDefault(locale.forLanguageTag("en-US")) # Set a default local
```

Comment:
Hmmm .. @viirya. Actually, wouldn't we better make it working? Seems like if we have default locales not available in JVM, it always fails (not only the test but this API itself).

So, looks always they have to manually change system locale or using this current way via accessing a private property `_jvm` in PySpark.

Wouldn't we maybe better just fallback to US locale by default with a warning?

For instance .. 

```scala
  private val getDefaultOrUS: String = {
    if (Locale.getAvailableLocales.map(_.toString).contains(Locale.getDefault.toString)) {
      Locale.getDefault.toString
    } else {
      logWarning(s"Default locale set was [${Locale.getDefault.toString}]; however, it was " +
        "not found in available locales in JVM, falling back to es-US locale. Set locale " +
        "in order to respect another locale.")
      Locale.US.toString
    }
  }
  setDefault(stopWords -> StopWordsRemover.loadDefaultStopWords("english"),
    caseSensitive -> false, locale -> getDefaultOrUS)
```

Reply:
Hmm, yeah, I agree, thro I just wanted to make the test work at first. Suggested way changes current behavior, but seems to me more reasonable.